examples/cifar10/pt/utils/cifar10_data_splitter.py

```diff
@@ -101,14 +91,25 @@ def split(self, fl_ctx: FLContext):
             site_file_name = site_file_path + str(site + 1) + ".npy"
             np.save(site_file_name, np.array(site_idx[site]))
 
-    def load_cifar10_data(self):
+    @staticmethod
+    def load_cifar10_data():
         # download data
         train_dataset = datasets.CIFAR10(root=CIFAR10_ROOT, train=True, download=True)
 
         # only training label is needed for doing split
         train_label = np.array(train_dataset.targets)
         return train_label
 
+    @staticmethod
+    def get_site_class_summary(train_label, site_idx):
+        class_sum = {}
+
+        for site, data_idx in site_idx.items():
+            unq, unq_cnt = np.unique(train_label[data_idx], return_counts=True)
+            tmp = {int(unq[i]): int(unq_cnt[i]) for i in range(len(unq))}
+            class_sum[site] = tmp
+        return class_sum
+
```

Comment:
Since both `cifar10_data_splitter.py` and `cifar10_vertical_data_splitter.py` is using these functions.

I prefer moving these methods out inside a file let's call it "data_utils.py" or "cifar10_data_utils.py".

And both of those classes import these utility methods from that file.

By doing this we don't need to import the class CIFAR10_Data_splitter in CIFAR10 vertical data splitter.

Reply:
Good idea. Changed.